tests/resources/test_bills.py

```diff
@@ -59,6 +60,21 @@ def test_cancel_bill(client):
     assert updated_transaction.status == 'refunded'
 
 
+@pytest.mark.vcr
+def test_cancel_bill_fail(client):
+    transactions = client.transactions.list()
+    transactions = list(filter(lambda t: t.status == 'fulfilled',
+                               transactions))
+    if transactions:
+        with pytest.raises(UnprocessableEntity) as excinfo:
+            transactions[0].cancel()
+        assert excinfo.value.code == 'R26'
+        assert excinfo.value.message == ('Reversal '
```

Comment:
try to avoid wrapping mid-string when possible. This could also be written as:
```python
assert excinfo.value.message == (
    'Reversal not supported for this biller')
```